### PR TITLE
🐛 avoid rclone getting killed

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

rclone is getting killed, most likely because it is using too much memory. This tries to fix the issue.

```
2023/02/22 05:21:33 INFO  :           0 B / 24.548 GiB, 0%, 0 B/s, ETA - (xfr#0/283)
2023/02/22 05:21:38 INFO  :   333.980 MiB / 24.548 GiB, 1%, 28.958 MiB/s, ETA 14m16s (xfr#0/283)
Killed
```

**NOTE:** the agent's service is limited to 200MB of RAM currently. Will try not to raise this value to see if we can manage to have a smaller rclone footprint. @mguidon this might be interesting to observer for future applications.

### CODE REVIEW

File: services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py [hunk 1]
```diff
@@ -176,6 +176,10 @@ async def store_to_s3(  # pylint:disable=too-many-locals,too-many-arguments
         f"{s3_retries}",
         "--transfers",
         f"{s3_parallelism}",
+        # limiting memory usage https://forum.rclone.org/t/how-to-set-a-memory-limit/10230/4
+        "--use-mmap",
```
Comment (sanderegg) [inline services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:180]: Could you comment on what these values are doing?

**Review by sanderegg — APPROVED** [review]

File: services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py (hunk 1 above)
Comment (GitHK) [reply services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:180]: They are telling go to use memory differently, this helps with freeing it faster.

Comment (sanderegg) [reply services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:180]: ok this is still not very helpful. what means differently?
Maybe just add comments in there as to there description and reference to the correct rclone documentation page.

Comment (GitHK) [reply services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py]: Let's see if now it looks better

**Review by odeimaiz — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**